assets/lang/en_US.ini

```diff
@@ -471,8 +457,6 @@ Use UI background = Use UI background
 % of the void = % of the void
 % of viewport = % of viewport
 %, 0:unlimited = %, 0 = unlimited
-(supersampling) = (supersampling)
-(upscaling) = (upscaling)
```

Comment:
These are still used in FramebufferManagerCommon:

https://github.com/hrydgard/ppsspp/blob/287dd66f20f2a03acd316a6f5d5b98ff2eb6ad26/GPU/Common/FramebufferManagerCommon.cpp#L2004-L2006

-[Unknown]